docs/migration/migrate_7_3.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[breaking-changes-7.3]]
+== Breaking changes in 7.3
+++++
+<titleabbrev>7.3</titleabbrev>
+++++
+
+This section discusses the changes that you need to be aware of when migrating
+your application to Kibana 7.3.
+
+See also <<release-highlights>> and <<release-notes>>.
+
+* <<breaking_73_dashboard_import_export>>
```

Comment:
Because there is only one list item, and users don't need to scroll to view the text, we probably don't need this bullet.